### ISSUE TITLE
Daily List is out of sync with Clicks

### ISSUE THREAD

**Issue by GalateaDreamer** [HabitRPG/habitica#5896]
For about a week now, my daily list has been out of sync when I click it. I'm not sure sync is the right word to use here, so I'll explain. Say I have my list as this:
Feed cats
Do Dishes
Take Medications

Well, I will click to complete the first two ("feed cats" and "do dishes"), and they will be completed. But when I get the "take medications", it will uncheck "feed cats" instead. The order is not exactly as set as it sounds here. Sometimes Its much more chaotic, but this is the idea. No amount of clicking, refreshing, or remaking the list helps.

**Comment by Alys** [comment]
We'll need to know details about your account before we can help with this. Can you please tell us your User ID? If you don't know it, go Help -> Report a Bug on the https://habitica.com/ website and see step 6.

**Comment by GalateaDreamer** [comment]
Oh sorry. User ID is 97c39652-b00b-4d21-a0d5-a89bd06c77dc. And I'm on Windows 8

**Comment by deilann** [comment]
Actually, we can't identify your account through your display name -- they
aren't unique. There could be another GalateaDreamer.

Please go here: https://github.com/HabitRPG/habitrpg/issues/2760 and follow
the steps under 6 to provide your UserID.

On Mon, Sep 7, 2015 at 11:00 AM, GalateaDreamer notifications@github.com
wrote:

> Oh sorry. I'm GalateaDreamer. And I am on Windows 8.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/HabitRPG/habitrpg/issues/5896#issuecomment-138350028.

## 

Currently Reading
https://www.goodreads.com/user/email_signature_destination/35215676?utm_medium=reading_link&utm_source=email_signature
[image: Book Cover]
https://www.goodreads.com/user/email_signature_destination/35215676?utm_medium=cover&utm_source=email_signature

**Comment by aclonegeek** [comment]
I started having this issue today but solved it by clearing my cache. You can do this by pressing CTRL + F5 while on Habitica (or completely clear your cache if that doesn't work).

**Comment by Alys** [comment]
@GalateaDreamer I've checked your account and you don't have any duplicate task IDs, which could have been one cause of this. It probably is just an issue with your browser being glitchy. If clearing your cache alone didn't solve it, please log out of Habitica, clear your cache and cookies, clear local storage (that's different than your cache - you might need to google for instructions for your browser), and close all browser windows. 

Please tell us if the problem is still happening or is fixed.

Just a note about GitHub: Whenever you make a new issue or add a new comment to an existing issue, we get emailed. However if you edit an old comment, we don't get emailed. I happened to notice that you had added your User ID only by chance. :) When you have any updates for a problem, putting them in a new comment is best.

**Comment by Alys** [comment]
I'm assuming this has stopped happening, so I'm closing this issue.

@GalateaDreamer If it's still happening, please see my previous comment.
